src/test/resources/features/meteringLabels.feature

```diff
@@ -0,0 +1,9 @@
+@check-metering-labels
```

Comment:
Just one more thing: the same tag is in syndesis-qe tests. They must by different because syndesis will run also this test.
Change it to @checkMeteringLabels, @check-apicurito-metering-labels or something suitable.

Also please add tag @apicuritoTests to be consistent with the rest od the test suite.